double-conversion/string-to-double.cc

```diff
@@ -146,22 +151,10 @@ static double SignedZero(bool sign) {
 
 
 // Returns true if 'c' is a decimal digit that is valid for the given radix.
-//
```

Comment:
I'm guessing this is not true anymore for newer vscode versions.
could we just change the `#ifdef` to include the version of the compiler?